.github/workflows/ci.yml

```diff
@@ -63,8 +63,45 @@ jobs:
       - name: run integration tests
         run: ./scripts/integration.sh
 
+  gpu-containers:
+    runs-on: [ self-hosted, gpu ]
```

Comment:
self-hosted? Maybe i'm not up to date with this task. Do we have self-hosted runners?

Reply:
Yes. Right now a test one (although it is full featured), after this PR I will create a scalable deployment. Go to `Settings > Actions > Runners` in this repo to see it :)